package.json

```diff
@@ -1,18 +1,19 @@
 {
   "name": "sfdx-codescan-plugin",
   "description": "Run CodeScan or SonarQube jobs from sfdx",
-  "version": "1.0.4",
+  "version": "1.0.5",
   "author": "Ben van Klinken @ustramooner",
-  "bugs": "https://github.com/VillageChief/sfdx-codescan-plugin/issues",
+  "bugs": "https://github.com/omazhar/sfdx-codescan-plugin/issues",
```

Comment:
Don't replace repo name with your fork `omazhar`

Reply:
certainly, i just wasn't sure how leaving the repo as is, might impact any downstream processes on your end